tests/regression/test_to_expression.py

```diff
@@ -0,0 +1,44 @@
+from firedrake import *
+import firedrake.expression as expression
+import pytest
+import numpy as np
+
+
+def test_to_expression_1D():
+    mesh = UnitIntervalMesh(5)
+    V = FunctionSpace(mesh, "CG", 1)
+
+    v1 = expression.to_expression(0.)
+    v2 = expression.to_expression([0.])
+    v3 = expression.to_expression("0.")
+    v4 = expression.to_expression(["0."])
+
+    f1 = interpolate(v1, V)
+    f2 = interpolate(v2, V)
+    f3 = interpolate(v3, V)
+    f4 = interpolate(v4, V)
+
```

Comment:
I didn't know that functions are initialized to zero. The goal of the test
is to check that you can write the same value in different fashions and
that all do the same thing. I'll add a line to test that they do the right thing.